hadoop-ozone/dist/src/main/compose/xcompat/test.sh

```diff
@@ -74,29 +75,21 @@ _read() {
     compatibility/read.robot
 }
 
-test_bucket_encryption() {
-  _kinit
-  execute_robot_test ${container} -N "xcompat-cluster-${cluster_version}-client-${client_version}" -v SUFFIX:${client_version} security/bucket-encryption.robot
```

Comment:
Can we delete this robot file now?

Reply:
Yes, I was also thinking about that.  Since `xcompat` also exercises "current client" with "current cluster", testing the same in `ozonesecure` is redundant.  (We can apply this to other functional tests later.)

Reply:
Done.